examples/windowmediaplayer.py

```diff
@@ -83,6 +80,22 @@ def windows_media():
     time.sleep(.5)
 
     app.ChooseColumsn.Cancel.click()
+    time.sleep(.5)
+
+    # Open a media file
+    app.WindowsMediaPlayer.menu_select("File->Open")
+
+    # Select and play a song
+    song_path = "#push your path"
+    window = app.Open
+    window.Wait('ready')
+    edit = window.Edit
+    edit.type_keys(song_path)
+    button = window[u'&Open']
+    button.Click()
+
+    # Let the music plays
+    time.sleep(20)

```

Comment:
This `time.sleep` is OK, of course. 👍 Maybe just need to align it with chosen track length.